src/preprocessor/PreprocessorParser.ts

```diff
@@ -251,10 +251,10 @@ export class PreprocessorParser {
     }
 
     private peek() {
-        return this.tokens[this.current];
+        return this.tokens![this.current];
     }
 
     private previous() {
-        return this.tokens[this.current - 1];
+        return this.tokens![this.current - 1];
```

Comment:
Why do you need the `!` here when it's initialized at the top of the file during construction?

Reply:
I'm assuming that I added the assertions before adding the default value and then forgot to go back to clean up. Nice catch!